incubator/java-spring-boot2/image/Dockerfile-stack

```diff
@@ -11,30 +11,27 @@ RUN  apt-get -qq update \
 COPY ./project /project
 COPY ./mvn-stack-settings.xml /usr/share/maven/conf/settings.xml
 COPY ./mvn-stack-settings.xml /project/.mvn-stack-settings.xml
-COPY ./java-spring-boot2-build.sh /project/java-spring-boot2-build.sh
 
-WORKDIR "/project"
-
-RUN mkdir -p /project/target && chmod 777 /project/target
+WORKDIR /project/user-app
 
 ENV APPSODY_USER_RUN_AS_LOCAL=true
 
-ENV APPSODY_MOUNTS="src:/project/src;pom.xml:/project/pom.xml;~/.m2/repository:/mvn/repository"
+ENV APPSODY_MOUNTS=".:/project/user-app;~/.m2/repository:/mvn/repository"
 ENV APPSODY_DEPS=
 
-ENV APPSODY_WATCH_DIR="/project"
-ENV APPSODY_WATCH_IGNORE_DIR="/project/target"
+ENV APPSODY_WATCH_DIR="/project/user-app/src"
+ENV APPSODY_WATCH_IGNORE_DIR=""
 ENV APPSODY_WATCH_REGEX=""
```

Comment:
Would you want to use any filters like only watch for changes to *.java files?

Reply:
no. Spring has other resource files that should also be watched.